db/migrate/0_database_structure.rb

```diff
@@ -0,0 +1,433 @@
+# encoding: UTF-8
+class FsekStructure < ActiveRecord::Migration
+  def change
+    unless table_exists? :album_categories
+      create_table "album_categories", force: :cascade do |t|
+        t.string "name", limit: 255
+        t.text "text", limit: 65535
+        t.boolean "visible", limit: 1
+        t.datetime "created_at"
+        t.datetime "updated_at"
+      end
+    end
+    unless table_exists? :album_categories_albums
+      create_table "album_categories_albums", id: false, force: :cascade do |t|
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.